src/views/wiki.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div class="wiki">
+        <h1 class="title">List of wiki articles (under construction)</h1>
+        <div class="structure-container">
+            <div v-for="category in structure">
+                <h2 class="category-title">{{ category.name }}</h2>
+                <div v-for="article in category.articles" class="article clickable"
+                     @click="onArticleClick(category, article)">
+                    {{ article.name }}
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'Wiki',
+    data() {
+        return {
+            structure: [
+                {
+                    name: 'Fields',
+                    folderName: 'fields',
+                    articles: [
+                        {name: 'Artifact', fileName: 'artifact'},
+                        {name: 'Control Point', fileName: 'control'},
```

Comment:
Detail: We might better name this article ``Control Field`` due to the context here. (explaining all types of special fields)